build/__snapshots__/features-meta.json

```diff
@@ -788,6 +788,11 @@
 		"description": "Preserves current branch and path when viewing all commits by an author.",
 		"screenshot": "https://user-images.githubusercontent.com/44045911/148764372-ee443213-e61a-4227-9219-0ee54ed832e8.png"
 	},
+	{
+		"id": "scroll-to-current-file",
+		"description": "Scrolls the file tree to the current file when signed in.",
```

Comment:
We don't care about the signed out experience, so no need to mention "signed in" here